src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/AlertSetting/GetAzureRmRecoveryServicesAsrAlertSetting.cs

```diff
@@ -29,7 +29,7 @@ namespace Microsoft.Azure.Commands.RecoveryServices.SiteRecovery
         "Get-ASRNotificationSetting",
         "Get-AzureRmRecoveryServicesAsrNotificationSetting",
         "Get-ASRAlertSetting")]
-    [OutputType(typeof(IEnumerable<ASRAlertSetting>))]
+    [OutputType(typeof(ASRAlertSetting))]
```

Comment:
@twitchax I would do a quick sweep over RecoveryServices.SiteRecovery to make sure everything is OK